src/main/java/frc/robot/subsystems/SwerveDriveSubsystem.java

```diff
@@ -105,10 +110,13 @@ public void periodic() {
 
         ntOdometryPose.setString(odometry.getPose().toString());
 
-        // ntBackLeftAngleEncoder.setDouble(backLeft.getEncoderPosition());
+        ntBackLeftAngleEncoder.setDouble(backLeft.getEncoderPosition());
         ntBackRightAngleEncoder.setDouble(backRight.getEncoderPosition());
-        // ntFrontLeftAngleEncoder.setDouble(frontLeft.getEncoderPosition());
-        // ntFrontRightAngleEncoder.setDouble(frontRight.getEncoderPosition());
+        ntFrontLeftAngleEncoder.setDouble(frontLeft.getEncoderPosition());
+        ntFrontRightAngleEncoder.setDouble(frontRight.getEncoderPosition());
+
+
+
 
         ntVelocityBackRight.setDouble(backRight.getSpeedMotor().getEncoder().getVelocity());
```

Comment:
Might be a good idea to change this to calling a new function from the WheelSubsystem called just `getVelocity` or something, and get rid of the speedMotor getter if this is the only reason you need it. Generally not a good idea to allow other classes access to an entire object just to access a single variable.